src/models/stores/app-config-model.ts

```diff
@@ -108,6 +108,17 @@ export const AppConfigModel = types
     getDocumentLabel(docType: string, num?: number, lowerCase?: boolean) {
       const docLabel = self.documentLabels[docType];
       return docLabel ? DocumentLabelModel.create(docLabel).getLabel(num, lowerCase) : "";
+    },
+    get showAnnotationControls() {
+      return (
+
+        // Controls are enabled by any setting of 'annotation' in the config other than 'none'
+        // in the future there will be more options supported.
+        self.annotations && self.annotations !== 'none' ||
```

Comment:
I think some parentheses would help clarify how to interpret this logic - a && b || c